test/unit/actions-test.js

```diff
@@ -79,6 +81,48 @@ describe('Denali.Action', function() {
     });
   });
 
+  describe.only('an app with a default JSON-API serializer', function() {
+    it('should not invoke the default serializer if serializer=false in an action', function() {
+      let success = false;
+      let mock = mockReqRes({
+        request: {
+          headers: {
+            'content-type': 'application/json',
+            'transfer-encoding': 'chunked',
+          },
+          method: 'post',
+          url: '/a-path',
+          body: { my: 'post-data' }
+        }
+      });
+      class IssuesAction extends Action {
+        serializer = false;
+        run() {
+          success = true;
+          expect(success).to.be.true();
```

Comment:
Yeah, the `success = true` stuff felt pretty ugly when I did it, thanks for suggestion 1, that helps.

Re suggestion 2, I'm feeling dense or something atm 😀 I tried using a `handle().then()` chain the other day and wasn't being sent back a promise.  I've updated `Router#handle()` per what I think you're saying, but would love further input / feedback, as I don't think I'm understanding what you're after ...